reports/Michigan.md

```diff
@@ -129,6 +129,20 @@ id: mi-detroit-9
 * https://twitter.com/DJEazyTwist/status/1277430912653950976
 
 
+### Riot police officer chokes protestor. Tear gas was also used. | July 11th
+
+A Police officer in full riot gear is seen arresting Detroit Will Breathe organizer, Nakia Renne Wallace by Chokehold. It was reported another individual was arrested by force with a knee to the neck. Nonviolent protestors were dispersed using shields, batons, and tear gas.
+
+tags: tear-gas, protestor, shield, baton, arrest, choke, knee-on-neck, tear-gas, tear-gas-canister  
```

Comment:
This is a bit of a nitpick, but I would remove the knee-on-neck tag since we only have antecdotal evidence of it.